oomd/root.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/adrg/xdg"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"gopkg.in/yaml.v2"
```

Comment:
shall we use yaml.v3?

Reply:
It works well in this scenario that I don't see the need to replace

Reply:
I mean, we're using yaml.v3 in apply. It'd better to use a same version across the codebase

Reply:
Makes sense, featctl also uses yaml.v2, so I replace it with another PR for uniformity?

Reply:
np